lookup.js

```diff
@@ -144,7 +144,7 @@ exports.init = function (sbot, config) {
 
     pull(
       sbot.db.query(
-        where(and(authorIsBendyButtV1(), isPublic())),
+        where(and(authorIsBendyButtV1(), isDecrypted())),
```

Comment:
@mixmix I think we can just remove `isPublic()` without adding `isDecrypted()` (context: we used to have "isPrivate()" but we split that into two operators: `isEncrypted()` and `isDecrypted()` and then we re-added `isPrivate()` as an alias for `or(isEncrypted(), isDecrypted())`)

If that doesn't work, then you can try explicitly:

```js
and(authorIsBendyButtV1(), or(isPublic(), isDecrypted()))
```